samples/utility.js

```diff
@@ -27,6 +27,28 @@
         console.log(gl.getShaderInfoLog(fshader));
 
         return program;
+    }
+
+    window.loadTexture = function(gl, src, callback) {
```

Comment:
I get why this was added, but I would not do this since it fragments the WebGL calls across files in a non-obvious way.  `createProgram`, for example, is much better isolated as is `loadImage`, which I would use in this case.